realm-transformer/build.gradle

```diff
@@ -1,14 +1,18 @@
 buildscript {
+    ext.kotlin_version = '1.0.1-2'
```

Comment:
Sneaking in Kotlin through the backdoor? :)
Personally I don't have any problems with it, but we probably shouldn't mix Kotlin/Groovy if we can avoid it. We should at least have team consensus if that is the direction we want to go in for the Transform API's.